core/src/com/unciv/UncivGame.kt

```diff
@@ -98,6 +98,12 @@ class UncivGame(
                 autoLoadGame()
                 thread(name="Music") { startMusic() }
                 isInitialized = true
+                if (Gdx.app.type == Application.ApplicationType.Desktop && settings.windowState.height>39 && settings.windowState.width>39) {
```

Comment:
What does these magic numbers `39` mean?

Reply:
Nothing really. Only a window smaller than 40x40 is not worth restoring, and to guard for the after-upgrade case where they're all zero. And 42 seemed too.. well-trodden.